mars/deploy/oscar/rayconfig.yml

```diff
@@ -11,7 +11,9 @@ cluster:
   node_timeout: 120
   node_check_interval: 1
   ray:
-    supervisor_exclusive_node: no
+    supervisor:
+      standalone: no
+      sub_pool_num: 0
```

Comment:
`sub_pool_num` should be specified in `mars.deploy.oscar.ray.new_cluster`

Reply:
IMHO, config.yml should be considered first.

Reply:
If the options in `config.yml` overrides the options in API, then we do not need to provide the same option in `mars.deploy.oscar.ray.new_cluster`. The `sub_pool_num` in `new_cluster` API will be overridden by the value in the default config `rayconfig.yml`.

We can refine the API in the future PR.